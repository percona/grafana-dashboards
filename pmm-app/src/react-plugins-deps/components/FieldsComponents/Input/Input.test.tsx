// Just a stub test
describe('placeholder test', () => {
  it('should return true', () => {
    expect(true).toBeTruthy();
  });
});
