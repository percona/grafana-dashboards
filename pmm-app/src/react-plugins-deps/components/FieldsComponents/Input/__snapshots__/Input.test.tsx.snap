// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Input field test Input renders correct without props 1`] = `
<div
  className="input-field-wrapper"
  style={Object {}}
>
  <input
    className="input-field input-field--dark"
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    style={Object {}}
    type="text"
    value=""
  />
</div>
`;

exports[`Input field test Input renders correct without props 2`] = `null`;
